ui/raidboss/data/04-sb/dungeon/fractal_continuum_hard.js

```diff
@@ -141,5 +265,40 @@
       netRegexKo: NetRegexes.startsUsing({ id: ['27AF', '27B0'], source: '알테마 비스트', capture: false }),
       response: Responses.getIn(),
     },
+    {
+      id: 'Fractal Hard Allagan Gravity',
+      netRegex: NetRegexes.startsUsing({ id: '27B5', source: 'The Ultima Beast', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '27B5', source: 'Ultimativ(?:e|er|es|en) Bestie', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '27B5', source: 'Ultima-Monstre', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '27B5', source: 'アルテマビースト', capture: false }),
+      netRegexCn: NetRegexes.startsUsing({ id: '27B5', source: '究极神兽', capture: false }),
+      netRegexKo: NetRegexes.startsUsing({ id: '27B5', source: '알테마 비스트', capture: false }),
+      response: Responses.spread(),
+    },
+    {
+      id: 'Fractal Hard Demi Ultima',
+      netRegex: NetRegexes.startsUsing({ id: '27B2', source: 'The Ultima Beast', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '27B2', source: 'Ultimativ(?:e|er|es|en) Bestie', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '27B2', source: 'Ultima-Monstre', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '27B2', source: 'アルテマビースト', capture: false }),
+      netRegexCn: NetRegexes.startsUsing({ id: '27B2', source: '究极神兽', capture: false }),
+      netRegexKo: NetRegexes.startsUsing({ id: '27B2', source: '알테마 비스트', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Fractal Hard Allagan Flare',
+      netRegex: NetRegexes.headMarker({ id: '0057' }),
+      alertText: function(data, matches) {
+        if (data.me === matches.target) {
+          return {
+            en: 'Flare on YOU',
```

Comment:
What do you think about `Responses.awayFrom()` here?

Reply:
That's definitely better, thank you!